src/content/configuration/optimization.md

```diff
@@ -40,27 +40,42 @@ T> Learn how [mode](/concepts/mode/) works.
 
 ## `optimization.minimizer`
 
-`[TerserPlugin]`
+`array` of `<plugin>` and/or `function`
```

Comment:
this is not the right way (to be consistent with the rest of the docs):
```suggestion
`[<plugin>]` and or `[function (compiler)]`
```

Reply:
that is if other plugins are supported if not, better to go with `[TerserPlugin]`

Reply:
Yes pointing to master makes total sense, please re-aplly PR fix that was lost

Reply:
@EugeneHlushko ok, all set